sdk/test/Services/S3/UnitTests/Custom/EmbeddedResource/property-aliases.json

```diff
@@ -0,0 +1,10 @@
+{
+  "PropertyAliases": {
```

Comment:
.net's property values are slightly different than whats in the mapping so i made another mapping to the .net values. the reason i didnt modify the original mapping.json is so we can copy and paste it from the SEP code directly if we ever need to update it